releasenotes/notes/ad_aks_ignore_pause_containers-024d32ddbc62d1ad.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+other:
+  - |
+    Ignore AKS pause containers hosted in the aksrepos.azurecr.io
+    container registry.
```

Comment:
```suggestion
---
enhancements:
  - |
    Ignore AKS pause containers hosted in the aksrepos.azurecr.io
    container registry.
```

Reply:
Thanks for the suggestion, I've updated my commit acccordingly!

Reply:
I don't see it, did you push the changes?